NavigateU/NavigateU/HomeScreen/Views/HomeDesignView.swift

```diff
@@ -6,33 +6,42 @@
 //
 
 import UIKit
+import SnapKit
 
-class HomeDesignView: UIView {
+final class HomeDesignView: UIView {
```

Comment:
there is no need to create an class for views, that will not be reused. The main idea of creating individual classes for views - is to reuse them in different screens of your app, for example, some button, that will look similar in whole project.

You can move all this views to view controller